sbi/analysis/plot.py

```diff
@@ -976,3 +978,353 @@ def _get_default_opts():
         },
         "title_format": {"fontsize": 16},
     }
+
+
+def sbc_rank_plot(
```

Comment:
this has a lot of arguments. I guess that most of them will not be relevant to most users. We could have a hidden `_sbc_rank_plot()` which takes all arguments a user-facing `sbc_rank_plot()` which supports only the most important arguments and then passes the unimportant arguments to `_sbc_rank_plot()` as `kwargs`.